src/core/providers/ogr/qgsogrproviderconnection.cpp

```diff
@@ -140,9 +356,30 @@ void QgsOgrProviderConnection::setDefaultCapabilities()
 
   char **driverMetadata = GDALGetMetadata( hDriver, nullptr );
 
+  mCapabilities = Capability::SqlLayers
+                  | Capability::ExecuteSql
+                  | Capability::Tables;
+
   if ( !CSLFetchBoolean( driverMetadata, GDAL_DCAP_NONSPATIAL, false ) && CSLFetchBoolean( driverMetadata, GDAL_DCAP_VECTOR, false ) )
     mCapabilities |= Capability::Spatial;
 
+#if GDAL_VERSION_NUM >= GDAL_COMPUTE_VERSION(13,4,0)
```

Comment:
```suggestion
#if GDAL_VERSION_NUM >= GDAL_COMPUTE_VERSION(3,4,0)
```

Reply:
😂 oops, that was my testing to make sure the older code path worked properly